org.libredirect.frontends-manager.yml

```diff
@@ -0,0 +1,52 @@
+id: org.libredirect.frontends-manager
+runtime: org.gnome.Platform
+runtime-version: '42'
+command: libredirect-frontends-manager
+rename-icon: libredirect-frontends-manager
+
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+  - org.freedesktop.Sdk.Extension.rust-nightly
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-data
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: libredirect-frontends-manager
+    sources:
+      - type: git
+        url: https://github.com/libredirect/frontends_manager.git
+        branch: main
+        tag: v1.0.0
+        commit: 1ab80c9068698db809151392ef2c9178d95d7dc7
+        x-checker-data:
+          type: git
+          tag-pattern: ^v((?:\d+.)*\d+)$
+      - type: file
+        path: org.libredirect.frontends-manager.desktop
```

Comment:
you build system build a debian package without a .desktop file?

Reply:
I'm just using the debian build bundle bc it's the most straight forward way to compile a binary from tauri.

Reply:
I would expect that this have a .desktop file. After all this is a 20+ year old standard and shipping an app without one is a bug. And if there is one why isn't it used.

Reply:
Alright removed the desktop file from this repo.